docs/update.md

```diff
@@ -290,12 +288,12 @@ spec:
   image: docker.io/percona/percona-postgresql-operator:{{ release }}-upgrade
   fromPostgresVersion: 15
   toPostgresVersion: 16
-  toPostgresImage: docker.io/percona/percona-postgresql-operator:{{ release }}-ppg{{ postgres16recommended }}-postgres
+  toPostgresImage: docker.io/percona-distribution-postgresql:{{ postgres16recommended }}
```

Comment:
Wrong image. Should be "docker.io/percona/percona-distribution-postgresql:{{ postgresrecommended }}",